api/src/main/java/keywhiz/api/model/Secret.java

```diff
@@ -47,6 +47,7 @@
   /** Base64-encoded content of this version of the secret. */
   private String secret;
   private final LazyString encryptedSecret;
+  private final String hmac;
```

Comment:
I'll update the PR to do this (forgot IntelliJ's "refactor" option would make this trivial).